themes/edx.org/lms/templates/dashboard.html

```diff
@@ -180,6 +180,19 @@ <h2 class="account-status-title sr">${_("Account Status Info")}: </h2>
       </ul>
     </section>
   </section>
+  % if xseries_credentials:
```

Comment:
I'm saying that I think it is incorrect to have XSeries/xseries written into the template names, method names and logic of the system.  While *all* operators of Open edX may choose to use Programs, edX is the *only* operator of Open edX which uses Programs of the category "xseries".  So I think references to this specific category should be factored out of the code and into confguration.  For example, we should not have a method named "_get_xseries_credentials()", it should be a more generic method that either takes the category name as an argument or pulls the category name in from configuration.